packages/components/src/index.ts

```diff
@@ -132,6 +132,7 @@ export {
 	NavigatorToParentButton as __experimentalNavigatorToParentButton,
 	useNavigator as __experimentalUseNavigator,
```

Comment:
Are we planning to deprecate the legacy ones at some point? Perhaps in a separate PR after we update the existing usages?

Reply:
Yup! From the "Next steps" section in the PR description:

> - refactor existing usages in Gutenberg to stable version
> - deprecate experimental version (both in JSDocs and with a runtime warning) (+ add unit tests for deprecation warnings)

Reply:
Thanks for confirming 😉 It's a long-going PR, so it never hurts to confirm.